# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/embutils/hotfix-dd-uclibc.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- embutils/dd.c.vanilla	2021-10-07 00:20:35.483157501 +0200
+++ embutils/dd.c	2021-10-07 00:20:48.939156965 +0200
@@ -3,7 +3,8 @@
 #include <fcntl.h>
 #include <sys/mman.h>
 #include <sys/shm.h>
-
+#include <sys/types.h>
+#include <unistd.h>
 #include <sys/uio.h>
 
 #ifndef PAGE_SIZE
